pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -79,6 +86,9 @@ var (
 		"pilot_k8s_endpoints_pending_pod",
 		"Number of endpoints that do not currently have any corresponding pods.",
 	)
+
+	enableEventing = env.RegisterBoolVar("K8S_EVENTS", true,
```

Comment:
nit: move to the features package

Reply:
I'm not a big fan of 'features' package - for temporary experiments I think it's cleaner to just keep them as private as possible, and 'features' really should be exposed in MeshConfig. We have tools to find all env settings I think.